test/testing.sh

```diff
@@ -577,7 +577,6 @@ fail_config_system_config() { rm -f "$testing_lrprefix/etc/luarocks/config.lua";
 test_config_system_config() { mkdir -p "$testing_lrprefix/etc/luarocks"; touch "$testing_lrprefix/etc/luarocks/config.lua"; $luarocks config --system-config; err=$?; rm -f "$testing_lrprefix/etc/luarocks/config.lua"; return $err; }
 fail_config_system_config_invalid() { mkdir -p "$testing_lrprefix/etc/luarocks"; echo "if if if" > "$testing_lrprefix/etc/luarocks/config.lua"; $luarocks config --system-config; err=$?; rm -f "$testing_lrprefix/etc/luarocks/config.lua"; return $err; }
 test_config_user_config() { $luarocks config --user-config; }
-fail_config_user_config() { LUAROCKS_CONFIG="/missing_file.lua" $luarocks config --user-config; }
```

Comment:
Why is this test removed? For some reason can't see it in the travis log on master either.